certified-connectors/Witivio/apiDefinition.swagger.json

```diff
@@ -1,1198 +1,1292 @@
-        "summary": "Get the schema of the webhook response",
-        "operationId": "GetWebhookResponseSchema",
-        "parameters": [
-          {
-            "name": "botId",
-            "in": "path",
-            "required": true,
-            "type": "string"
-          },
-          {
-            "name": "questionId",
-            "in": "path",
-            "required": true,
-            "type": "string"
-          },
-          {
-            "name": "language",
-            "in": "path",
-            "required": true,
-            "type": "string"
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "OK",
-            "schema": {
-              "type": "object"
+                }
             }
-          }
-        }
-      }
-    },
-    "/api/conversations/{botId}/{language}/questions/{questionId}/triggers/register": {
-      "x-ms-notification-content": {
-        "schema": {
-          "type": "object",
-          "x-ms-dynamic-schema": {
-            "operationId": "GetWebhookResponseSchema",
-            "parameters": {
-              "botId": {
-                "parameter": "botId"
-              },
-              "questionId": {
-                "parameter": "questionId"
-              },
-              "language": {
-                "parameter": "language"
-              }
-            },
-            "value-path": "properties"
-          }
-        },
-        "description": "Webhook response"
-      },
-      "post": {
-        "responses": {
-          "201": {
-            "description": "Created"
-          }
         },
-        "summary": "When the chatbot understand",
-        "operationId": "WebhookTrigger",
-        "description": "",
-        "x-ms-trigger": "single",
-        "parameters": [
-          {
-            "name": "licenceId",
-            "in": "query",
-            "required": true,
-            "type": "string",
-            "description": "Select the licence.",
-            "x-ms-summary": "Select the licence",
-            "x-ms-dynamic-values": {
-              "operationId": "GetLicences",
-              "value-path": "id",
-              "value-title": "clientName"
-            }
-          },
-          {
-            "name": "botId",
-            "in": "path",
-            "required": true,
-            "type": "string",
-            "description": "Select the chatbot you want to trigger from.",
-            "x-ms-summary": "Select the chatbot",
-            "x-ms-dynamic-values": {
-              "operationId": "GetChatbotIDs",
-              "parameters": {
-                "licenceId": {
-                  "parameter": "licenceId"
+        "/api/bot/{licenceId}/my": {
+            "get": {
+                "x-ms-visibility": "internal",
+                "description": "Get your chatbots",
+                "summary": "Get your chatbots",
+                "operationId": "GetChatbotIDs",
+                "parameters": [
+                    {
```

Comment:
@tdechanterac Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is longer tab index between the properties. Please refer the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.